src/routes/auth/login.route.js

```diff
@@ -0,0 +1,50 @@
+const jwt = require('jsonwebtoken')
+const bcrypt = require('bcryptjs')
+const crypto = require('crypto')
+const User = require('../../entities/user')
+
+module.exports.post = {
+	method: 'POST',
+	path: '/login',
+	schema: {
+		body: {
+			type: 'object',
+			additionalProperties: false,
+			required: ['username', 'password'],
+			properties: {
+				username: {
+					type: 'string',
+				},
+				password: {
+					type: 'string',
+				},
+			},
+		},
+	},
+	handler: async (req, res, _next) => {
+		const { username, password } = req.body
+
+		const user = await User.findOne({ username })
+		if (!user) {
+			return res.status(400).json({ message: 'Invalid username or password' })
+		}
+
+		const passwordMatch = await bcrypt.compare(password, user.password)
+		if (!passwordMatch) {
+			return res.status(400).json({ message: 'Invalid username or password' })
+		}
+		
+		const refreshToken = crypto.randomBytes(64).toString('hex')
+		user.refreshToken = refreshToken
```

Comment:
`refreshToken` is currently not being saved to the user even though mongo doesn't complain. That's because we're still using a "table schema" which you can find in `src/entities/user.js`. Adding the field `refreshToken: String` solves the problem and the token is saved to the user